### DISCUSSION TITLE
S3 Minio Error  InvalidAccessKeyId

### DISCUSSION THREAD

**Discussion by yildirima** [outline/outline#2236] [Self Hosting]
You have documented that outline supports minio but I have been stuck with a ridiculous problem while configuring outline to use minio.

**To Reproduce**
1-Installed outline on Kubernetes and  it's working without any problem.
2. Define configuration for minio for S3
3. Try to insert any image on a document.
4. Get error in POST request (403)

#outline Config Map

data:
  PORT: '8080'
  AWS_S3_FORCE_PATH_STYLE: 'true'
  REDIS_URL: 'redis://redis-cluster:6379'
  DATABASE_URL: 'postgres://wikiusr:pass@wiki-postgresql:5432/wiki'
  AWS_S3_ACL: private
  TEAM_LOGO: 'https://test.com/png'
  ENABLE_UPDATES: 'true'
  URL: 'https://wiki.test.com'
  GOOGLE_ALLOWED_DOMAINS: wiki.test.com
  DEBUG: 'cache,presenters,events,emails,mailer,utils,multiplayer,server,services'
  AWS_REGION: istanbul
  DEFAULT_LANGUAGE: en_US
  FORCE_HTTPS: 'true'
  PGSSLMODE: disable
  GOOGLE_ANALYTICS_ID: ''
  AWS_S3_UPLOAD_BUCKET_URL: 'https://minio.apps.internal.com/'
  AWS_S3_UPLOAD_MAX_SIZE: '26214400'
  AWS_S3_UPLOAD_BUCKET_NAME: test

#outline BASE64 Secrets: ( minio default secret and  access key used. minio/minio123) I can use them while access bucket.
data:
  SLACK_VERIFICATION_TOKEN: base64SomeSecret
  SLACK_APP_ID: base64SomeSecret
  GOOGLE_CLIENT_ID: base64SomeSecret
  MINIO_ACCESS_KEY: bWluaW8K
  SECRET_KEY: >-
    base64SomeSecret==
  SLACK_KEY: base64SomeSecret
  AWS_SECRET_ACCESS_KEY: bWluaW8xMjMK
  AWS_ACCESS_KEY_ID: bWluaW8K
  MINIO_SECRET_KEY: bWluaW8xMjMK
  UTILS_SECRET: >-
    base64SomeSecret=
  GOOGLE_CLIENT_SECRET: base64SomeSecret
  SLACK_SECRET: base64SomeSecret=

When  I  tried to upload any image, I got this error.  The other secrets works but I think there is a problem about Minio integration. 
<Error>
<Code>
InvalidAccessKeyId
</Code>
<Message
>
The Access Key Id you provided does not exist in our records.
</Message>
<BucketName>
test
</BucketName>
<Resource>
/test
</Resource>
<RequestId>
168969F89F53E90D
</RequestId>
<HostId>
5f641805-623c-4174-bbdd-f205d8df0789
</HostId>
</Error>

* Minio hots ls output

test:
  URL       : http://minio:9000
  AccessKey : minio
  SecretKey : minio123
  API       : s3v4
  Path      : auto

**Comment by yildirima** [comment]
Also I tried to use play.minio.io for the S3 service got same error.

**Reply by tommoor** [reply]
Screenshot the request in your devtools and share, it's probably posting to not quite the right location on minio – bucket missing or something like that.
